app/controllers/api/v1/deliveries_controller.rb

```diff
@@ -57,6 +57,12 @@ def update
       end
     end
 
+    api :DELETE, '/v1/deliveries/1', "Delete delivery"
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.